### PR TITLE
feat: Get Enrollments API

### PR DESCRIPTION
**Pull request by ajeet-now**

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -7,5 +7,5 @@ var router = express.Router();
 
 router.get('/:id',auth , user_controller.get_user_by_id)
 router.put('/:id',auth , user_controller.update_user_by_id)
-
+router.get('/enrollments/:id', auth, user_controller.get_user_enrollments)
```
Comment (ssi259) [inline routes/users.js:10]: if we are passing user_id we should keep the variable as user_id.also pass it as query params

File: services/user_manager.js [hunk 2]
```diff
@@ -36,4 +36,27 @@ exports.process_update_user_by_id = async (input_data, req) => {
...
+}
+
+
+exports.pre_process_get_user_enrollments = async (req) => {
+    console.log("user_id",req.params.id)
+    return req.params.id
+}
+
+exports.process_get_user_enrollments = async (user_id) => {
+    console.log("user_id 1",user_id)
```
Comment (ssi259) [inline services/user_manager.js:48]: remove logging here

File: services/user_manager.js [hunk 3]
```diff
@@ -36,4 +36,27 @@ exports.process_update_user_by_id = async (input_data, req) => {
...
+    const enrollments = await models.Enrollment.findAll({
+        where: {
+            user_id: user_id,
+            status:"active"
```
Comment (ssi259) [inline services/user_manager.js:52]: in UI there can be a section to display user's previous enrollements.we should handle the status field either through params or for now we can pass all the enrollments of any user and filteration of active can be done on UI.discuss with arihant if needs only active enrollments

Comment (ajeet-now) [reply services/user_manager.js:52]: right

File: services/user_manager.js (hunk 2 above)
Comment (ajeet-now) [reply services/user_manager.js:48]: sure

**Pull request merged**